# HG changeset patch
# Parent 828d5f7388d2413ac76c3880771994b6e23bd3d9

diff -r 828d5f7388d2 lib/Target/ARM/ARMTargetObjectFile.cpp
--- a/lib/Target/ARM/ARMTargetObjectFile.cpp	Tue Jul 26 13:38:23 2011 -0700
+++ b/lib/Target/ARM/ARMTargetObjectFile.cpp	Tue Jul 26 13:38:23 2011 -0700
@@ -36,7 +36,7 @@
                                  ELF::SHF_WRITE |
                                  ELF::SHF_ALLOC,
                                  SectionKind::getDataRel());
-    LSDASection = NULL;
+    // LSDASection = NULL; @LOCALMOD
   }
 
   AttributesSection =
