# HG changeset patch
# Parent 1c7e433109fe6bf796cfd291d23dad3f3a7a8883

diff -r 1c7e433109fe tools/lto/LTOModule.cpp
--- a/tools/lto/LTOModule.cpp	Wed Jul 06 17:30:45 2011 -0700
+++ b/tools/lto/LTOModule.cpp	Wed Jul 06 17:30:45 2011 -0700
@@ -141,8 +141,8 @@
   }
 
   // parse bitcode buffer
-  OwningPtr<Module> m(getLazyBitcodeModule(buffer, getGlobalContext(),
-                                           &errMsg));
+  OwningPtr<Module> m(ParseBitcodeFile(buffer, getGlobalContext(), // @LOCALMOD
+                                       &errMsg));
   if (!m) {
     delete buffer;
     return NULL;
