# HG changeset patch
# Parent 674a1768f66e2507c1cf83cd14114685e8dc8184

diff -r 674a1768f66e lib/Target/ARM/ARMTargetObjectFile.cpp
--- a/lib/Target/ARM/ARMTargetObjectFile.cpp	Wed Jul 27 11:11:48 2011 -0700
+++ b/lib/Target/ARM/ARMTargetObjectFile.cpp	Wed Jul 27 11:11:49 2011 -0700
@@ -36,7 +36,7 @@
                                  ELF::SHF_WRITE |
                                  ELF::SHF_ALLOC,
                                  SectionKind::getDataRel());
-    LSDASection = NULL;
+    // LSDASection = NULL; @LOCALMOD
   }
 
   AttributesSection =
