# HG changeset patch
# Parent 8c795c4390f5d20a3ba1e3f26b12f75f2aca4765

diff -r 8c795c4390f5 tools/lto/LTOModule.cpp
--- a/tools/lto/LTOModule.cpp	Thu Jul 14 15:24:55 2011 -0700
+++ b/tools/lto/LTOModule.cpp	Thu Jul 14 15:24:55 2011 -0700
@@ -141,8 +141,8 @@
   }
 
   // parse bitcode buffer
-  OwningPtr<Module> m(getLazyBitcodeModule(buffer, getGlobalContext(),
-                                           &errMsg));
+  OwningPtr<Module> m(ParseBitcodeFile(buffer, getGlobalContext(), // @LOCALMOD
+                                       &errMsg));
   if (!m) {
     delete buffer;
     return NULL;
