# HG changeset patch
# Parent 3b05ab40a0cafb53a8a3324851835e4704f2bf5d

diff -r 3b05ab40a0ca tools/lto/LTOModule.cpp
--- a/tools/lto/LTOModule.cpp	Thu Jul 28 16:24:52 2011 -0700
+++ b/tools/lto/LTOModule.cpp	Thu Jul 28 16:24:52 2011 -0700
@@ -141,8 +141,8 @@
   }
 
   // parse bitcode buffer
-  OwningPtr<Module> m(getLazyBitcodeModule(buffer, getGlobalContext(),
-                                           &errMsg));
+  OwningPtr<Module> m(ParseBitcodeFile(buffer, getGlobalContext(), // @LOCALMOD
+                                       &errMsg));
   if (!m) {
     delete buffer;
     return NULL;
