# HG changeset patch
# Parent be237e11df4031ad5917a473a21014719ffe5a9b

diff -r be237e11df40 tools/lto/LTOModule.cpp
--- a/tools/lto/LTOModule.cpp	Mon Jul 11 14:29:18 2011 -0700
+++ b/tools/lto/LTOModule.cpp	Mon Jul 11 14:29:19 2011 -0700
@@ -140,8 +140,8 @@
   }
 
   // parse bitcode buffer
-  OwningPtr<Module> m(getLazyBitcodeModule(buffer, getGlobalContext(),
-                                           &errMsg));
+  OwningPtr<Module> m(ParseBitcodeFile(buffer, getGlobalContext(), // @LOCALMOD
+                                       &errMsg));
   if (!m) {
     delete buffer;
     return NULL;
