# HG changeset patch
# Parent 93c41562c89f9e51e74daced973393afbf26a83b

diff -r 93c41562c89f tools/lto/LTOModule.cpp
--- a/tools/lto/LTOModule.cpp	Tue Jul 26 13:38:23 2011 -0700
+++ b/tools/lto/LTOModule.cpp	Tue Jul 26 13:38:24 2011 -0700
@@ -141,8 +141,8 @@
   }
 
   // parse bitcode buffer
-  OwningPtr<Module> m(getLazyBitcodeModule(buffer, getGlobalContext(),
-                                           &errMsg));
+  OwningPtr<Module> m(ParseBitcodeFile(buffer, getGlobalContext(), // @LOCALMOD
+                                       &errMsg));
   if (!m) {
     delete buffer;
     return NULL;
