# HG changeset patch
# Parent ac1f00daa6365116ea0991cd2f2a77f8d6eac74e

diff -r ac1f00daa636 tools/lto/LTOModule.cpp
--- a/tools/lto/LTOModule.cpp	Wed Jul 27 14:41:46 2011 -0700
+++ b/tools/lto/LTOModule.cpp	Wed Jul 27 14:41:47 2011 -0700
@@ -141,8 +141,8 @@
   }
 
   // parse bitcode buffer
-  OwningPtr<Module> m(getLazyBitcodeModule(buffer, getGlobalContext(),
-                                           &errMsg));
+  OwningPtr<Module> m(ParseBitcodeFile(buffer, getGlobalContext(), // @LOCALMOD
+                                       &errMsg));
   if (!m) {
     delete buffer;
     return NULL;
