# HG changeset patch
# Parent 3164433e8b509f17d78af3d2a60d58c501687a10

diff -r 3164433e8b50 tools/lto/LTOModule.cpp
--- a/tools/lto/LTOModule.cpp	Fri Jul 29 14:32:51 2011 -0700
+++ b/tools/lto/LTOModule.cpp	Fri Jul 29 14:32:51 2011 -0700
@@ -141,8 +141,8 @@
   }
 
   // parse bitcode buffer
-  OwningPtr<Module> m(getLazyBitcodeModule(buffer, getGlobalContext(),
-                                           &errMsg));
+  OwningPtr<Module> m(ParseBitcodeFile(buffer, getGlobalContext(), // @LOCALMOD
+                                       &errMsg));
   if (!m) {
     delete buffer;
     return NULL;
