# HG changeset patch
# Parent f309af4d786f39c418349571f08c2c0ccd327027

diff -r f309af4d786f tools/lto/LTOModule.cpp
--- a/tools/lto/LTOModule.cpp	Fri Jul 29 11:21:31 2011 -0700
+++ b/tools/lto/LTOModule.cpp	Fri Jul 29 11:21:32 2011 -0700
@@ -141,8 +141,8 @@
   }
 
   // parse bitcode buffer
-  OwningPtr<Module> m(getLazyBitcodeModule(buffer, getGlobalContext(),
-                                           &errMsg));
+  OwningPtr<Module> m(ParseBitcodeFile(buffer, getGlobalContext(), // @LOCALMOD
+                                       &errMsg));
   if (!m) {
     delete buffer;
     return NULL;
