# HG changeset patch
# Parent 399f3591b12d08a887e839ebe3631052d763255e

diff -r 399f3591b12d lib/Target/ARM/ARMTargetObjectFile.cpp
--- a/lib/Target/ARM/ARMTargetObjectFile.cpp	Thu Jul 28 16:24:51 2011 -0700
+++ b/lib/Target/ARM/ARMTargetObjectFile.cpp	Thu Jul 28 16:24:51 2011 -0700
@@ -36,7 +36,7 @@
                                  ELF::SHF_WRITE |
                                  ELF::SHF_ALLOC,
                                  SectionKind::getDataRel());
-    LSDASection = NULL;
+    // LSDASection = NULL; @LOCALMOD
   }
 
   AttributesSection =
