# HG changeset patch
# Parent 52582aeabd7f8f5fa1123144c506094ef1b28484

diff -r 52582aeabd7f lib/Target/ARM/ARMTargetObjectFile.cpp
--- a/lib/Target/ARM/ARMTargetObjectFile.cpp	Mon Jul 11 09:37:43 2011 -0700
+++ b/lib/Target/ARM/ARMTargetObjectFile.cpp	Mon Jul 11 09:37:44 2011 -0700
@@ -36,7 +36,7 @@
                                  ELF::SHF_WRITE |
                                  ELF::SHF_ALLOC,
                                  SectionKind::getDataRel());
-    LSDASection = NULL;
+    // LSDASection = NULL; @LOCALMOD
   }
 
   AttributesSection =
