# HG changeset patch
# Parent 937fee6bb58b3af9b1716a05aee30816990d1fbe

diff -r 937fee6bb58b lib/Target/ARM/ARMTargetObjectFile.cpp
--- a/lib/Target/ARM/ARMTargetObjectFile.cpp	Mon Jul 11 14:29:17 2011 -0700
+++ b/lib/Target/ARM/ARMTargetObjectFile.cpp	Mon Jul 11 14:29:18 2011 -0700
@@ -36,7 +36,7 @@
                                  ELF::SHF_WRITE |
                                  ELF::SHF_ALLOC,
                                  SectionKind::getDataRel());
-    LSDASection = NULL;
+    // LSDASection = NULL; @LOCALMOD
   }
 
   AttributesSection =
