# HG changeset patch
# Parent 9da29a89af8f3138c0cb69a076774ef644f62b82

diff -r 9da29a89af8f tools/lto/LTOModule.cpp
--- a/tools/lto/LTOModule.cpp	Wed Jul 27 11:11:49 2011 -0700
+++ b/tools/lto/LTOModule.cpp	Wed Jul 27 11:11:50 2011 -0700
@@ -141,8 +141,8 @@
   }
 
   // parse bitcode buffer
-  OwningPtr<Module> m(getLazyBitcodeModule(buffer, getGlobalContext(),
-                                           &errMsg));
+  OwningPtr<Module> m(ParseBitcodeFile(buffer, getGlobalContext(), // @LOCALMOD
+                                       &errMsg));
   if (!m) {
     delete buffer;
     return NULL;
