# HG changeset patch
# Parent e557403a56298d076c06a298029cf65e86c85f0c

diff -r e557403a5629 lib/Target/ARM/ARMTargetObjectFile.cpp
--- a/lib/Target/ARM/ARMTargetObjectFile.cpp	Fri Jul 29 14:32:50 2011 -0700
+++ b/lib/Target/ARM/ARMTargetObjectFile.cpp	Fri Jul 29 14:32:50 2011 -0700
@@ -36,7 +36,7 @@
                                  ELF::SHF_WRITE |
                                  ELF::SHF_ALLOC,
                                  SectionKind::getDataRel());
-    LSDASection = NULL;
+    // LSDASection = NULL; @LOCALMOD
   }
 
   AttributesSection =
