# HG changeset patch
# Parent 83fecca32559350679d094c17cb6694d7ce3468f

diff -r 83fecca32559 tools/llc/llc.cpp
--- a/tools/llc/llc.cpp	Fri Jul 29 14:32:51 2011 -0700
+++ b/tools/llc/llc.cpp	Fri Jul 29 14:32:52 2011 -0700
@@ -245,7 +245,7 @@
   FOS.flush();
   ROS.flush();
 
-#if defined(__native_client__)
+#if defined(__native_client__) && defined(NACL_SRPC)
   NaClOutputStringToFile(filename, s);
 #else
   std::string error;
