# HG changeset patch
# Parent f74c420406a47f7d6a7320afebbfb43d0c044538

diff -r f74c420406a4 tools/llc/llc.cpp
--- a/tools/llc/llc.cpp	Thu Jul 28 16:24:52 2011 -0700
+++ b/tools/llc/llc.cpp	Thu Jul 28 16:24:53 2011 -0700
@@ -245,7 +245,7 @@
   FOS.flush();
   ROS.flush();
 
-#if defined(__native_client__)
+#if defined(__native_client__) && defined(NACL_SRPC)
   NaClOutputStringToFile(filename, s);
 #else
   std::string error;
