# HG changeset patch
# Parent a1fdf914b21cf00272eaba5c4f362f2d4fb51efb

diff -r a1fdf914b21c tools/llc/llc.cpp
--- a/tools/llc/llc.cpp	Wed Jul 27 14:41:47 2011 -0700
+++ b/tools/llc/llc.cpp	Wed Jul 27 14:41:47 2011 -0700
@@ -245,7 +245,7 @@
   FOS.flush();
   ROS.flush();
 
-#if defined(__native_client__)
+#if defined(__native_client__) && defined(NACL_SRPC)
   NaClOutputStringToFile(filename, s);
 #else
   std::string error;
