# HG changeset patch
# Parent e92e3e18897211221315027feb185a14734f3744

diff -r e92e3e188972 tools/llc/llc.cpp
--- a/tools/llc/llc.cpp	Wed Jul 27 11:11:50 2011 -0700
+++ b/tools/llc/llc.cpp	Wed Jul 27 11:11:50 2011 -0700
@@ -245,7 +245,7 @@
   FOS.flush();
   ROS.flush();
 
-#if defined(__native_client__)
+#if defined(__native_client__) && defined(NACL_SRPC)
   NaClOutputStringToFile(filename, s);
 #else
   std::string error;
