# HG changeset patch
# Parent a08dba02b248df9517272947b3179cf419c79269

diff -r a08dba02b248 tools/llc/llc.cpp
--- a/tools/llc/llc.cpp	Tue Jul 26 13:38:24 2011 -0700
+++ b/tools/llc/llc.cpp	Tue Jul 26 13:38:25 2011 -0700
@@ -245,7 +245,7 @@
   FOS.flush();
   ROS.flush();
 
-#if defined(__native_client__)
+#if defined(__native_client__) && defined(NACL_SRPC)
   NaClOutputStringToFile(filename, s);
 #else
   std::string error;
