# HG changeset patch
# Parent 9801f964a5af512564c7415b4e93187f40069177

diff -r 9801f964a5af tools/llc/llc.cpp
--- a/tools/llc/llc.cpp	Fri Jul 29 13:13:44 2011 -0700
+++ b/tools/llc/llc.cpp	Fri Jul 29 13:13:45 2011 -0700
@@ -245,7 +245,7 @@
   FOS.flush();
   ROS.flush();
 
-#if defined(__native_client__)
+#if defined(__native_client__) && defined(NACL_SRPC)
   NaClOutputStringToFile(filename, s);
 #else
   std::string error;
